tahoma_api/client.py

```diff
@@ -14,16 +16,26 @@
 class TahomaClient:
     """ Interface class for the Tahoma API """
 
-    def __init__(self, username: str, password: str, api_url: str = API_URL) -> None:
+    def __init__(
+        self,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        api_url: str = API_URL,
```

Comment:
Is it really something important? Do you know other endpoints?

Reply:
There's an issue opened in our component for that. https://github.com/iMicknl/ha-tahoma/issues/85